src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -1119,6 +1119,16 @@ export class PostgresStorageAdapter implements StorageAdapter {
     this._notifySchemaChange();
   }
 
+  async updateFieldOptions(className: string, fieldName: string, type: any) {
+    await this._client.tx('update-schema-field-options', async t => {
```

Comment:
this is a strange mix of function callbacks and promises, could you perhaps use something such as:

```
const t = await new Promise(resolve => this._client.tx('update-schema-field-options', resolve));
```

Reply:
the Postgres code mainly use this kind of structure since the SDK was not completly migrated from callback to promise. here i prefer to stick to the current code style with a Promise/Callback style